.github/workflows/e2e-tests-preview.yml

```
@@ -0,0 +1,676 @@
+      - name: 🔍 Extract Preview URL
+        id: extract-url
+        run: |
+          if [ "${{ github.event_name }}" = "workflow_dispatch" ] && [ -n "${{ inputs.preview_url }}" ]; then
+            # Manual trigger with specific URL
+            PREVIEW_URL="${{ inputs.preview_url }}"
+            echo "preview-url=$PREVIEW_URL" >> $GITHUB_OUTPUT
+            echo "📍 Using provided preview URL: $PREVIEW_URL"
+          elif [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
+            # Manual trigger without URL - auto-discover from GitHub deployments
+            echo "🔍 Auto-discovering preview URL for current commit..."
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)